### PR TITLE
Schedule session assessment

### PR DESCRIPTION
**Pull request by lionsdwarf**

### CODE REVIEW

File: apps/portal/pages/schedule-session.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ class ScheduleSessionPage extends React.Component {
     this.setState({
       step: this.state.step + 1,
       calendarUrl: clinician.calendarUrl,
-      disabledMsg: 'Please finish scheduling first'
+      // disabledMsg: 'Please finish scheduling first'
```
Comment (craigspaeth) [inline apps/portal/pages/schedule-session.js]: I was having trouble today with the polling code working too. I [made note](https://trello.com/c/m8sB9slx/186-look-into-alternative-to-polling-on-schedule-step-b-c-of-cliniko-rate-limiting) that we'll probably have to do something more low-tech.

**Comment by craigspaeth** [comment]
LGTM! Good luck with merging my work to create the—now 7 step 😱—scheduling flow.

**Comment by lionsdwarf** [comment]
@craigspaeth Here's our 7-step flow!

**Comment by craigspaeth** [comment]
Solid! LGTM, thanks!

**Pull request merged**